src/GovUk.Education.ExploreEducationStatistics.Content.Requests/GovUk.Education.ExploreEducationStatistics.Content.Requests.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Content.Model\GovUk.Education.ExploreEducationStatistics.Content.Model.csproj" />
+    <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Content.Services\GovUk.Education.ExploreEducationStatistics.Content.Services.csproj" />
```

Comment:
Shouldn't need a dependency on `Content.Services`. We can remove this by moving the `PublicationsSortBy` enum into this subproject.

Reply:
Good spot, I've switched the dependency around